sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -177,7 +177,7 @@ class RocksDB(
         numKeysOnWritingVersion -= 1
       }
     }
-    writeBatch.remove(key)
+    writeBatch.delete(key)
```

Comment:
The `remove` series method of `WriteBatch` is identified as `@Deprecated` in 6.x and [deleted in 7.x](https://github.com/facebook/rocksdb/pull/9576), here use `delete` instead 

Am I right @adamretter ？

Reply:
Yes.

Reply:
thanks @adamretter